### PR TITLE
Fix/extract context string value

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue https://github.com/telefonicaid/iotagent-node-lib/issues/1066

### CODE REVIEW

**Comment by AlvaroVega** [comment]
attributeList = [{"name":"bat_v","type":"string","value":"4.56"},{"name":"status","type":"string","value":"ok"},{"name":"cont1","type":"string","value":"232"},{"name":"t1","type":"string","value":"1"},{"name":"t2","type":"string","value":"240"},{"name":"datedat","type":"string","value":"2021-06-02T10:21:53.631Z"}]
[
  { name: 'bat_v', type: 'string', value: '4.56' },
  { name: 'status', type: 'string', value: 'ok' },
  { name: 'cont1', type: 'string', value: '232' },
  { name: 't1', type: 'string', value: '1' },
  { name: 't2', type: 'string', value: '240' },
  {
    name: 'datedat',
    type: 'string',
    value: '2021-06-02T10:21:53.631Z'
  }
]


extractContext(attributeList)
{
  bat_v: 4.56,
  status: 'ok',
  cont1: 232,
  t1: 1,
  t2: 240,
  datedat: '2021-06-02T10:21:53.631Z'
}

File: lib/plugins/jexlParser.js [hunk 1]
```diff
@@ -53,20 +53,23 @@ jexl.addTransform('urldecode', (val) => decodeURI(val));
...
+                (acc, curr, i, arr) => (acc === 0 || acc ? acc : val <= curr ? (acc = i) : (acc = null)),
+                null
+            )
+        ]
 );
-jexl.addTransform(
-    'bitwisemask',
-    (i,mask,op) => op==="&"?parseInt(i)&mask: op==="|"?parseInt(i)|mask: op==="^"?parseInt(i)^mask:i
+jexl.addTransform('bitwisemask', (i, mask, op) =>
+    op === '&' ? parseInt(i) & mask : op === '|' ? parseInt(i) | mask : op === '^' ? parseInt(i) ^ mask : i
```
Comment (fgalan) [inline lib/plugins/jexlParser.js:70]: Need to be sync in https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/expressionLanguage.md#available-functions or they are just indent/linter changes?

Comment (AlvaroVega) [reply lib/plugins/jexlParser.js:70]: just indent / linter auto changes

Comment (fgalan) [reply lib/plugins/jexlParser.js:70]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**